client/app/telecom/telephony/billingAccount/translations/Messages_fr_FR.xml

```diff
@@ -11,5 +11,5 @@
 
    <translation id="telephony_group_loading_error" qtlid="334205">Impossible de charger les informations relatives à votre compte de facturation ({{billingAccount}}).</translation>
    <translation id="telephony_group_rename_error" qtlid="334218">Impossible de renommer votre compte de facturation {{billingAccount}}.</translation>
-   <translation id="telephony_group_termination_warning" qtlid="357265">Le groupe sera supprimé le <strong>{{ date }}</strong></translation>
+   <translation id="telephony_group_termination_warning">Le groupe sera supprimé le <strong>{{ date }}</strong>. <a data-ui-sref="telecom.telephony.deleteGroup">Vous pouvez annuler en cliquant ici</a>.</translation>
```

Comment:
should take `<a ui-sref>` out of the trad, it will be easier to handle route changes later